### PR TITLE
tlmgr-gui: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/common/tlmgr-gui.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# tlmgr gui
+
+> Start the graphical user interface of `tlmgr`.
```
Comment (CleanMachine1) [inline pages/common/tlmgr-gui.md:3]:
```suggestion
> Start a graphical user interface of `tlmgr`.
```

Comment (CleanMachine1) [reply pages/common/tlmgr-gui.md:3]: If you agree with this change, make sure to basically apply it to all of the command descriptions!

Comment (marchersimon) [reply pages/common/tlmgr-gui.md:3]: Hm, I'm not sure. You typically only start one instance at a time.

Comment (CleanMachine1) [reply pages/common/tlmgr-gui.md:3]: It feels to possessive, does it need the GUI or is there a terminal version I haven't seen due to my ignorance of the command

Comment (marchersimon) [reply pages/common/tlmgr-gui.md:3]: It's mainly a terminal program

Comment (CleanMachine1) [reply pages/common/tlmgr-gui.md:3]: What do you think of 
```suggestion
> Start a graphical user interface version of `tlmgr`.
```

Comment (sbrl) [reply pages/common/tlmgr-gui.md:3]: What about this?
```suggestion
> Start a graphical user interface for `tlmgr`.
```

Comment (marchersimon) [reply pages/common/tlmgr-gui.md:3]: Sounds good.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Pull request merged**